scripts/content.js

```diff
@@ -139,13 +139,13 @@ const observer = new MutationObserver((mutations, instance) => {
   fix();
 
   if (!isPreview) {
-    mutations.forEach((mutation) => {
-      mutation.addedNodes.forEach((node) => {
-        const valid = check(node);
-
-        if (valid && node.matches(selectors)) node.outerHTML = "";
-      });
-    });
+    const nodes = mutations.map((mutation) => [...mutation.addedNodes])
+                           .flat()
+                           .filter((node) => check(node));
+    
+    for (const node of nodes) {
```

Comment:
Benchmarks say that a `for of` loop is more performant than a `for each` loop in some browsers. How about changing these two `for each` by two `for of` loops?

Reply:
Yup, will do it!